### PR TITLE
Pagemetadata

### PR DESCRIPTION
**Pull request by arjun87743**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix https://adobe-acs-group.atlassian.net/browse/WRLDBAEMS-99

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://pagemetadata--world-bank--aemsites.hlx.live/

view-source:https://author-p136806-e1403562.adobeaemcloud.com/content/world-bank/global/en/drafts/arjun/sample2.html?ref=pagemetadata

### CODE REVIEW

File: component-models.json [hunk 1]
```diff
@@ -712,16 +712,327 @@
...
         "name": "jcr:title",
-        "label": "Page Title",
+        "label": "Title",
+        "required": true,
         "description": "Sets the title of the page in page meta."
       },
+      {
+        "component": "text",
+        "valueType": "string",
+        "name": "jcr:pagetitle",
```
Comment (arjun87743) [inline component-models.json:722]: "jcr:pageTitle" was throwing lint error hence needed to change the name to "jcr:pagetitle"

**Comment by arjun87743** [comment]
ContentType is set as tag rather than a dropdown.
Languages added are 7 as of now (2 are of Chinese ) instead of 180 languages.

File: component-models.json [hunk 2]
```diff
@@ -712,16 +712,327 @@
...
+        "required": true,
         "description": "Sets the description of the page in page meta."
       },
+      {
+        "component": "aem-tag",
+        "name": "content_type",
+        "label": "Content Type",
+        "valueType": "string",
+        "required": true,
+        "rootPath": "/content/cq:tags/world-bank/content-type"
```
Comment (mkbansal1) [inline component-models.json:740]: Change it to dropdown for now

File: component-models.json [hunk 3]
```diff
@@ -716,16 +716,417 @@
...
+            "name": "Leadership",
+            "value": "leadership"
+          },
+          {
+            "name": "Listing",
+            "value": "listing"
+          },
+          {
+            "name": "Loans & Credits",
+            "value": "loans-&-Credits"
```
Comment (thomsebastin) [inline component-models.json:792]: is this a casing error or requirement?

Comment (arjun87743) [reply component-models.json]: casing error, updated the PR

**Review by mkbansal1 — APPROVED** [review]

**Pull request merged**